waltz-common/src/main/java/com/wepay/waltz/common/message/MessageCodecV2.java

```diff
@@ -47,8 +47,8 @@ public Message decode(MessageAttributeReader reader) {
                 return new MountRequest(reqId, clientHighWaterMark, seqNum);
 
             case MessageType.MOUNT_RESPONSE:
-                boolean partitionReady = reader.readBoolean();
-                return new MountResponse(reqId, partitionReady);
+                int partitionState = reader.readInt();
```

Comment:
I think we have to update the MessageCodec version as we are changing the partitionState from Boolean to an Integer which means this wont be backward compatible.

Reply:
Yes. We need to create V3 to change the wire protocol.